### DISCUSSION TITLE
Is HB_OPTIMIZE_SIZE safe to use?

### DISCUSSION THREAD

**Discussion by sid-wr** [harfbuzz/harfbuzz#4923] [General] [answered]
Hi
I tried `HB_OPTIMIZE_SIZE` flag & am able to see a bit of compression.
Can you please confirm if this flag is safe to use? I mean would it impact any existing functionality?

**Comment by behdad** [comment, marked as answer]
Yes it's safe. Also see `CONFIG.md` for other size reduction options.
